Source/WebGPU/WebGPU/PresentationContextIOSurface.mm

```diff
@@ -194,7 +194,11 @@ static void generateAValidationError(Device& device, NSString* message, bool gen
     MTLTextureDescriptor *textureDescriptor = [MTLTextureDescriptor texture2DDescriptorWithPixelFormat:Texture::pixelFormat(effectiveFormat) width:width height:height mipmapped:NO];
     textureDescriptor.usage = Texture::usage(descriptor.usage, effectiveFormat);
 #if PLATFORM(MAC) || PLATFORM(MACCATALYST)
```

Comment:
@heycam confirmed macCatalyst should still use managed when the device does not support unified memory, otherwise an iOS app running on an Intel Mac with AMD graphics card will suffer considerable performance penalty syncing memory across the PCI bus